packages/server/src/service/wisconsinFipsCode.ts

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios'
```

Comment:
There seems to be a lot of overlap here with michiganFipsCode.  Perhaps we should just call this `queryArgGIS`.  We can parameterize by a method `getFipsCode(x: any)` and selects off the proper value (e.g. `.DOA` vs `.FIPSCODE`).